keras_nlp/layers/masked_lm_head.py

```diff
@@ -140,10 +140,10 @@ def __init__(
             self.vocabulary_size = shape[0]
 
     def build(self, input_shapes):
-        feature_size = input_shapes[-1]
+        embedding_dim = self.embedding_weights.shape[-1]
```

Comment:
nice catch on this! we should handle the case when `embedding_weights == None` too. something like

```
if self.embedding_weights:
    inner_dim = self.embedding_weights.shape[-1]
else:
    inner_dim = input_shapes[-1]
```

Reply:
we might also want to add a quick on the unit tests for this layer. copy `test_valid_call_with_embedding_weights`, but make the input dim != embedding dim